src/plugins/kibana/server/lib/schemas/resources/index_pattern_schema.js

```diff
@@ -6,6 +6,7 @@ module.exports = Joi.object({
   time_field_name: Joi.string(),
   interval_name: Joi.string(),
   not_expandable: Joi.boolean(),
+  fieldFilters: Joi.string().default('[]'),
```

Comment:
This should probably use underscores rather than camel case as well, to be consistent with the rest of the properties.